ui/src/app/components/spinners.tsx

```diff
@@ -31,6 +31,7 @@ const styles = reactStyles({
 export const Spinner = ({ dark = false, size = 72, style = {}, ...props }) => {
   return (
     <svg
+      aria-label='Please Wait'
```

Comment:
what do you think?

Reply:
I think it's better than before, but suspect we still have a [lot to learn](https://stackoverflow.com/questions/38704467/how-to-label-a-loading-animation-for-wai-aria).